pubsub/pubsubhub.go

```diff
@@ -123,7 +100,7 @@ func NewPubSubHub(dataSource wsDataSource, auxDataSource wsDataSourceAux) *PubSu
 	// Development subsidy address of the current network
 	devSubsidyAddress, err := dbtypes.DevSubsidyAddress(params)
 	if err != nil {
-		log.Warnf("NewPubSubHub: bad project fund address (%v)", err)
```

Comment:
So it was actually just a warning because in testnet2 there was no dev subsidy address, but we didn't want to break everything in dcrdata.  Does it hurt to ignore this error?  I don't expect a future testnet will have this problem, but who knows.

Reply:
There have been tests added to prevent that issue in the future, so I highly doubt it will be an issue.  Also, I'd rather just fix testnet out of the gate if there is some issue like that in the future instead of adding hacks in other software like dcrdata.

Reply:
Music to my ears.